change-beta/@azure-communication-react-8ea2d9dd-cc23-4c73-92fb-aeeaaa1467ec.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
nit: this is type none as we don't declare this internal-only change in the changelogs